downstream/modules/platform/proc-aap-migration.adoc

```diff
@@ -0,0 +1,7 @@
+[id="aap-migration_{context}"]
+
+= Migrating data to the {PlatformNameShort} Operator
+
+[role=_abstract]
+
+Once you have set your secret key, postgresql credentials, verified network connectivity and installed the {OperatorPlatform}, you can migrate your data by creating a custom resource controller object.
```

Comment:
```suggestion
.Prerequisites

* You have set your secret key. 
* You have set your postgresql credentials. 
* You have verified network connectivity. 
* You have installed the {OperatorPlatform}. 

.Procedure

. Migrate your data by creating a custom resource controller object. 
```
This suggestion is based on [Modular Documentation standard](https://redhat-documentation.github.io/modular-docs/)s for procedure modules. The question now is will the end user know how to create a custom resource controller object? Is this something that is assumed for average end user who will be following this procedure? Are there steps that are missing?